### PR TITLE
mod_admin: ensure that text search results are sorted on relevance

### PR DESCRIPTION
**Pull request by mworrell**
### Description

Fix a problem where the full text search in the admin is not sorted by relevance.

This changes the sorting of the full text queries in the admin to be on relevance, unless a sort column is clicked *after* the search has been done.

### Checklist

- [ ] documentation updated
- [ ] tests added
- [x] no BC breaks

### CODE REVIEW

File: modules/mod_admin/templates/_admin_menu.tpl [hunk 1]
```diff
@@ -80,7 +80,7 @@
 
             {% block search %}
                 <form class="navbar-right navbar-form form-inline" action="{% block search_target %}{% url admin_overview_rsc %}{% endblock %}" method="get">
-                    <input type="hidden" name="qsort" value="{{ q.qsort|escape }}" />
+                    {# <input type="hidden" name="qsort" value="{{ q.qsort|escape }}" /> #}
```
Comment (ddeboer) [inline modules/mod_admin/templates/_admin_menu.tpl]: Just remove this?

Comment (mworrell) [reply modules/mod_admin/templates/_admin_menu.tpl]: Ja, kan er nu wel uit (als dit gedrag goed genoeg is).

**Comment by mworrell** [comment]
@ddeboer did you also patch master?  Or shall I do that?

**Comment by ddeboer** [comment]
Please go ahead. 😄

**Comment by mworrell** [comment]
Done.

**Pull request merged**